tests/test_pytorch/test_dynamic_shapes.py

```diff
@@ -95,10 +97,6 @@ def calculate_auto_detect_total_recompile_counts(csv_file_path):
             print(f"Error: CSV file not found: {csv_file_path}")
             return None
 
-    # Close dist pg if initialized.
-    if torch.distributed.is_available() and torch.distributed.is_initialized():
```

Comment:
why removed ?

Reply:
This is moved as part of  standalone_only test, so pg is guaranteed to be not initialized there.